File: python/Calculator.py
class Calculator:
    def __init__(self) -> None:
        pass

    def addition(num1):
        print("Enter aditional number: ", num1, " + ");
        num2 = float(input());
        sum = num1 + num2;
        return sum;

    def subtraction(num1):
        print("Enter aditional number: ", num1, " - ");
        num2 = float(input());
        sub = num1 - num2;
        return sub;

    def multiplication(num1):
        print("Enter aditional number: ", num1, " * ");
        num2 = float(input());
        mul = num1 * num2;
        return mul;

    def division(num1):
        print("Enter aditional number: ", num1, " / ");
        num2 = float(input());
        div = num1 + num2;
        return div;


    def calculate(self):
        print("Enter number");
        num1 = float(input());
        itr = 1;      

        while(itr):            
            print("Enter operator: ", num1);
            opr = input();

            if(opr == '+'):                
                num1 = Calculator.addition(num1);
            elif(opr == '-'):
                num1 = Calculator.subtraction(num1);
            elif(opr == '*'):
                num1 = Calculator.multiplication(num1);
            elif(opr == '/'):
                num1 = Calculator.division(num1);
            print("Do you want to continue (1/0)");
            itr = int(input());

        print("Result: ", num1);
    

File: java/src/com/java/exercises/Calculator.java
package com.java.exercises;
import java.util.Scanner;

public class Calculator {
	public static void calculate() {
		try (Scanner input = new Scanner(System.in)) {
			System.out.println("Enter number");		
			float firstNum = input.nextFloat();
			int itrat = 1;
			
			while(itrat == 1){
				System.out.println("Enter operator ");	
				String oprat = input.next();
				char opr = oprat.charAt(0);
				if(opr == '+') {
					System.out.println("Enter another number " + firstNum + " " + opr);		
					float secNum = input.nextFloat();
					firstNum = addition(firstNum, secNum);
				}else if(opr == '-') {
					System.out.println("Enter another number "  + firstNum + " " + opr);		
					float secNum = input.nextFloat();
					firstNum = substraction(firstNum, secNum);
				}else if(opr == '*') {
					System.out.println("Enter another number "  + firstNum + " " + opr);		
					float secNum = input.nextFloat();
					firstNum = multiplication(firstNum, secNum);
				}else if(opr == '/') {
					System.out.println("Enter another number "  + firstNum + " " + opr);		
					float secNum = input.nextFloat();
					firstNum = division(firstNum, secNum);
				}
				System.out.println("To exit enter 0 to continue 1");	
				itrat = input.nextInt();
			}
			
			System.out.println(firstNum);
		}catch(Exception e) {
			System.out.println(e.getMessage());
		}
		
	}
	
	public static float addition(float num1, float num2) {
		float sum = num1 + num2;
		return sum;
	}
	
	public static float multiplication(float num1, float num2) {
		float product = num1 * num2;
		return product;
	}
	
	public static float substraction(float num1, float num2) {
		float sub = num1 - num2;
		return sub;
	}
	
	public static float division(float num1, float num2) {
		float quotient = num1 / num2;
		return quotient;
	}

}
